builds/checkin/compatibility-tool-checkin.yaml

```diff
@@ -0,0 +1,25 @@
+trigger: none
+pr:
+  branches:
```

Comment:
Instead of doing a git function call to check if the script file has changed, its better to include the check here

https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/sprint-194-update#post-neutral-status-to-github-when-a-build-is-skipped